content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md

```diff
@@ -143,15 +142,15 @@ If **Load Balancer** is selected, the following must be true:<p></p><br />
    - Load Balancer Source Ranges (Optional) - Limit which client IP's can access the load balancer. Inputs **must** be a comma-separated list of CIDR ranges in the `a.b.c.d/x` format. See [Network Load Balancer support on AWS](https://kubernetes.io/docs/concepts/services-networking/service/#aws-nlb-support) for additional details.
 <p></p><br />
 
-  ![Deploy-sandbox-cluster](deploy-nested-cluster.png "Deploy a Sandbox  Cluster")
+  ![Deploy-palette-virtual-cluster](deploy-nested-cluster.png "Deploy a Palette Virtual Cluster")
 
-Congratulations! Your Sandbox  Cluster is now deployed. If you like more information on managing your resources, refer to the links below.
+Congratulations! Your Palette Virtual Cluster is now deployed. If you like more information on managing your resources, refer to the links below.
```

Comment:
If you would like more information on managing your resources, see the links below.